pigpen-cascading/src/main/clojure/pigpen/cascading/core.clj

```diff
@@ -1,16 +0,0 @@
-(ns pigpen.cascading.core
```

Comment:
Oh, I thought it was mainly to combine stuff from several namespaces into one. My main reason to delete this is that Intellij does a much better job resolving stuff when you point to the original file. Not a very legit reason, I know.